assets/css/_ladder.scss

```diff
@@ -29,29 +29,29 @@
   cursor: pointer;
 
   &:hover {
-    .m-vehicle-icon__triangle {
+    .c-vehicle-icon__triangle {
       stroke: $color-secondary-light;
     }
 
-    .m-vehicle-icon__ghost-highlight {
+    .c-vehicle-icon__ghost-highlight {
       stroke: $color-secondary-light;
     }
 
-    .m-vehicle-icon__label-background {
+    .c-vehicle-icon__label-background {
       fill: $color-secondary-light;
     }
   }
 
   &.selected {
```

Comment:
**question:** in a few places we do have classnames like this that represent some sort of component state that don't fall into our overall naming scheme... should we do something with them or let that wait for another time?

Reply:
One point of view is that this is what the "M" in "BEM" is for, so it should actually be `.m-ladder__vehicle--selected`.

Reply:
I don't think we should handle it in this PR but I think it's reasonable to update it to match, maybe we just update them as we catch them?
Either way, I'll open a PR for this one

cc: @KaylaBrady

Reply:
https://github.com/mbta/skate/pull/2004